app/scripts/migrations/132.ts

```diff
@@ -0,0 +1,49 @@
+import { isObject } from '@metamask/utils';
+import { cloneDeep } from 'lodash';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 132;
+
+/**
+ * This migration sets `redesignedTransactionsEnabled` as true by default in preferences in PreferencesController.
+ *
+ * @param originalVersionedData - Versioned MetaMask extension state, exactly what we persist to dist.
+ * @param originalVersionedData.meta - State metadata.
+ * @param originalVersionedData.meta.version - The current state version.
+ * @param originalVersionedData.data - The persisted MetaMask state, keyed by controller.
+ * @returns Updated versioned MetaMask extension state.
+ */
+export async function migrate(
+  originalVersionedData: VersionedData,
+): Promise<VersionedData> {
+  const versionedData = cloneDeep(originalVersionedData);
+  versionedData.meta.version = version;
+  transformState(versionedData.data);
+  return versionedData;
+}
+
+function transformState(
+  state: Record<string, unknown>,
+): Record<string, unknown> {
+  const preferencesControllerState = state?.PreferencesController as
+    | Record<string, unknown>
+    | undefined;
```

Comment:
We have supported the case when no preferences state is found in previous migrations. It may be a good idea to support it here as well. Then we can ensure we set `redesignedTransactionsEnabled` for this migration. Maybe it's automatically set to true by default later, in which case we wouldn't need this. 

example: migration 122

```
  if (!isObject(state.PreferencesController?.preferences)) {
    state.PreferencesController = {
      preferences: {},
    };
  }
```